app/main.py

```diff
@@ -1,6 +1,15 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def inner(*args) -> Any:
```

Comment:
Please, use decorator `wraps` imported from functools for `inner` function. Read the purpose of this decorator.